### ISSUE TITLE
How do you markup breadcrumb using an icon or <abbr>

### ISSUE THREAD

**Issue by MrMerkin** [schemaorg/schemaorg#2949]
I just started learning JSON-LD and semantic markup in html using schema.org context. I setup a breadcrumb list, but I'm stumped on what to do for the first list item, because I'm using an icon for "Home." Both the JSON-LD and the html validate, but I can't get the markup validator to recognize the icon as "Home." I haven't found anything about this when searching.

This is setup using an icon font and instead of using a pseudo `::before` on a span or `<i>` tag, I tried using the character (in this case number `9`) inside of an `<abbr>` with a `title="Home"` attribute and used css to apply the `font-family`. Didn't work. I also tried wrapping it in a `<span>` but got the same result. Validator only sees the `9`.

What's the correct way to markup an icon link and link it to the ListItem in JASON-LD? Also curious how one would handle an `<abbr>` tag in general.

Code Example:
```jsonld
{
	"@context": "https://schema.org",
	"@type": "BreadcrumbList",
	"@id": "<?= $the_bread_link ?>#the-breadcrumbs",
	"itemListElement": [{
		"@type": "ListItem",
		"position": 1,
		"name": "Home",
		"item": "<?= $home_url ?>"
	},
```
```html
<ul class="l-breadcrumb" itemid="the-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
	<li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
		<a class="b-home" href="/" itemscope itemtype="http://schema.org/Thing" itemprop="item" itemid="/">
		<abbr title="Home" itemprop="name">9</abbr></a>&nbsp; &gt;
		<meta itemprop="position" content="1">
	</li>
```

**Comment by WeaverStever** [comment]
Try this?
~~~~
<abbr title="Home" itemprop="name" content="Home">9</abbr>
~~~~

A good example of the JSON-LD markup here:
https://developers.google.com/search/docs/advanced/structured-data/breadcrumb

**Comment by MrMerkin** [comment]
> Try this?
> 
> ```
> <abbr title="Home" itemprop="name" content="Home">9</abbr>
> ```
> 
> A good example of the JSON-LD markup here:
> https://developers.google.com/search/docs/advanced/structured-data/breadcrumb

Thank you, it worked! 

So the `content` attribute will act as an override. I've been using the Google docs but I'm reading as I work. I don't think I've come across HTML markup in their docs yet. They do have a lot of examples with JSON-LD. I used the page you linked to as a guide when doing this. If they showed an example of `content=` I must have missed it. 

Thanks for the help. Much appreciated.

**Comment by WeaverStever** [comment]
@MrMerkin 
We'd been discussing this elsewhere, should pass the WC3 validator but not others.  I could not find any documentation on content= (other than in meta).  I think it is the handle for what lies between the opening and closing tags. It seems to work everywhere.

Handy for writing empty, self closing, tags in microdata.

 ~~~~
 <span title="Homepage" itemprop="alternateName" content="Homepage" />
 ~~~~

**Comment by MrMerkin** [comment]
> @MrMerkin
> We'd been discussing this elsewhere, should pass the WC3 validator but not others. I could not find any documentation on content= (other than in meta). I think it is the handle for what lies between the opening and closing tags. It seems to work everywhere.
> 
> Handy for writing empty, self closing, tags in microdata.
> 
> ```
> <span title="Homepage" itemprop="alternateName" content="Homepage" />
> ```

Yeah, it seems to do the trick. I tested this in Google's `Rich Results Test` , `Schema Markup Validator`, and `W3C Nu Validator`, and it passed all 3 without errors after I adjusted the rest of my code.

I'm working with WordPress, so it's not all as straightforward since so much of the content is dynamic I can't hard-code everything. I have to load in a bunch of custom post fields and then filter some with PHP `strstr` and `html_entity_decode` to reformat variables specifically for JSON-LD. Then I echo those same things in the HTML markup in `itemid`. Took a lot of back-and-forth testing, but now everything validates.

Thanks again for the help and the tip. I'm not a programmer by trade and have learned 4 new languages for this project, so if my questions seem amateur, that's why ;)
